cmd/podman/networks/create.go

```diff
@@ -42,6 +42,8 @@ func networkCreateFlags(cmd *cobra.Command) {
 
 	flags.BoolVar(&networkCreateOptions.Internal, "internal", false, "restrict external access from this network")
 
+	flags.IntVar(&networkCreateOptions.MTU, "mtu", 0, "MTU")
```

Comment:
Should we do a more general options flag? It seems like there could be quite a lot of these.

Reply:
(The Docker extremely-long-option format is probably unnecessary, but something like `--bridge-option mtu=8000`?

Reply:
I suppose there couldn't be _many_ more than what the CNI "bridge" exposes, but hopefully the user (OP) has more input...

Reply:
In my use case I need only the MTU setting to sync it with an underlying network.
When looking at the bridge driver parameters, there's maybe one more good candidate for this - the `vlan` tag parameter..

Reply:
I agree with @mheon Lets do this as --bridge-option and just add the two for now, which leaves us room for more in the future without exploding out CLI.

Reply:
The other commands use "opt" (instead of option), for instance the one from "podman volume create"

```
  -o, --opt stringArray   Set driver specific options (default [])
```

Like the ones from "run":

```
      --dns-opt strings                          Set custom DNS options
      --log-opt strings                          Logging driver options
      --security-opt stringArray                 Security Options
```

Reply:
SGTM